src/Normalizer/NormalizerObjectMappingRegistry.php

```diff
@@ -42,6 +42,12 @@ private function addObjectMapping(NormalizationObjectMappingInterface $objectMap
      */
     public function getObjectMapping(string $class): NormalizationObjectMappingInterface
     {
+        $ref = new \ReflectionClass($class);
+
+        if (true === in_array('Doctrine\Common\Persistence\Proxy', $ref->getInterfaceNames(), true)) {
```

Comment:
@TiMESPLiNTER please do not use true === if the return value of a method is a boolean